app/containers/ViewProfilePage/Activity.jsx

```diff
@@ -19,7 +19,7 @@ import profileMessages from 'containers/Profile/messages';
 import QuestionsProfileTab from './QuestionsProfileTab';
 import Banner from './Banner';
 
-const DEFAULT_NUMBER = 10;
+const DEFAULT_NUMBER = 100000;
```

Comment:
I don't think that should change. I think we are showing only 10 items in the recent activity.